code/game/objects/items/crystal_destabilizer.dm

```diff
@@ -0,0 +1,15 @@
+/obj/item/crystal_destabilizer
+	name = "Supermatter Destabilizer"
+	desc = "Used for disrupting the fabric of the Crystal Matrix."
+	icon = 'icons/obj/supermatter.dmi'
+	icon_state = "destabilizer"
+	slot_flags = ITEM_SLOT_BACK
+	w_class = WEIGHT_CLASS_HUGE
+	///The destabilizer is one use only
+	var/filled = TRUE
+
+/obj/item/crystal_destabilizer/Initialize()
+	. = ..()
+
+/obj/item/crystal_destabilizer/Destroy()
+	return..()
```

Comment:
does this work

Reply:
well is not supposed to be destroyed, but i figure i should still add this in case an admin delete the item

Reply:
Then why override it if you're not changing the behavior?

Reply:
removed